### PR TITLE
Use ubuntu as base image, allow executing as current user

### PR DESCRIPTION
**Pull request by pladaria**
* Uses a 64bit ubuntu as base image. It is bulky but more similar to a regular linux desktop
* Changes to allow executing docker as current user. This way resulting files in ./out are not owned by root

To execute as current user:
`docker run --rm -v "$PWD":/src -u $(id -u):$(id -g) sgdk`

:heavy_check_mark: verified on Ubuntu Linux
:heavy_check_mark: verified on macOS Catalina 10.15.7

Also fixes: https://github.com/Stephane-D/SGDK/issues/217

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# https://EditorConfig.org
```
Comment (pladaria) [inline .editorconfig:1]: @Stephane-D what do you think?

This way everyone should use the same eol convention

**Comment by brunosxs** [comment]
much better than my solution of doing a chown in the entrypoint.

File: .editorconfig (hunk 1 above)
Comment (Stephane-D) [reply .editorconfig:1]: I think that is indeed better to have same EOL for everyone.

**Comment by Stephane-D** [comment]
I didn't tested it myself as i'm on Windows but from different comments it looks like i can safely apply the merge request !

**Pull request merged**